sns_ik_examples/src/ik_tests.cpp

```diff
@@ -41,6 +43,19 @@ double fRand(double min, double max)
   return min + f * (max - min);
 }
 
+bool in_vel_bounds(KDL::JntArray vals, KDL::JntArray vels)
```

Comment:
Sorry, I thought the comments defaulted to being inline. Here is where vals and vels were confusing.